nodeEditor/src/graphEditor.tsx

```diff
@@ -157,6 +157,11 @@ export class GraphEditor extends React.Component<IGraphEditorProps, IGraphEditor
             }
         });
 
+        this.props.globalState.onImportFrameObservable.add(() => {
+            this.loadGraph();

```

Comment:
why do we need to call this.loadGraph() ?

Reply:
I was using it to draw the nodes. refactored so now i am drawing nodes in GraphEditor.addFrameFromSerialization()

Reply:
you should be bale to ask for a repaint with globalData.refreshRequired if my memory serves

Reply:
I no longer need the repaint since I am calling createNodeFromObject() in GraphEditor.addFrameFromSerialization()